src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,9 @@
+.catalog {
+  padding-top: 50px;
+  display: flex;
+  flex-wrap: wrap;
+  max-width: 950px;
+  margin: 0 auto;
```

Comment:
you can't set margins for blocks, you can make new element and set margin on these element